.pre-commit-config.yaml

```diff
@@ -0,0 +1,25 @@
+repos:
```

Comment:
Here's the autostyle - all default really except for line-length 100.

The flake8 stuff I pulled from an existing project that I had already disabled annoying warnings for, feel free to debate but at least E501 (line-length) should definitely be off since we're handling that elsewhere (in black).